index.js

```diff
@@ -11,40 +11,37 @@ var PluginError   = require('plugin-error')
   , temp          = require('temp').track()
   , spawn         = require('cross-spawn')
   , Q             = require('q')
-  , defaultArgs   = ['--yes']
+  , defaultArgs   = ['make']
   , PLUGIN        = 'gulp-elm';
 
 function getDefaultExe() {
   try{
-    var elm_make_dev = path.resolve('node_modules/.bin/elm-make');
-    if(fs.statSync(elm_make_dev).isFile()){
-      return elm_make_dev;
+    var elm_dev = path.resolve('node_modules/.bin/elm');
+    if(fs.statSync(elm_dev).isFile()){
+      return elm_dev;
     }
   }catch (err){
     // local elm-make not available
   }
 
-  return 'elm-make';
+  return 'elm';
 }
 
-function processMakeOptions(options, output) {
+function processElmOptions(options, output) {
   var args   = defaultArgs
     , ext    = '.js'
     , exe    = getDefaultExe()
     , spawn  = {};
 
   if(!!options){
-    var yes = options.yesToAllPrompts;
-    if(yes !== undefined && !yes) {
-      args = [];
-    }
-
-    if(options.elmMake) { exe = options.elmMake; }
+    if(options.elm) { exe = options.elm; }
 
     if(options.warn === true) { args = args.concat(['--warn']); }

```

Comment:
`--warn` is not currently supported in 0.19 (though Evan said it may be added back, so maybe we just leave it in…)